src/pages/Categories/index.js

```diff
@@ -0,0 +1,5 @@
+import styles from './Categories.module.css';
+
+const CategoriesPage = () => <div className={styles.container}>Categories page</div>;
```

Comment:
- [ ] According to the requirements:

![image](https://user-images.githubusercontent.com/77069011/229014013-4b1f952a-16a7-4b95-919c-c0fa5451874c.png)

Please consider adding a button with the text "Check Status" on it! 🤓 🔧